sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/aio/_response_handlers_async.py

```diff
@@ -21,7 +21,7 @@ async def healthcare_extract_page_data_async(
             health_job_state.results
             if hasattr(health_job_state, "results")
             else health_job_state.tasks.items[0].results,
-            response_headers,
```

Comment:
curious is this a breaking change?

Reply:
No this code is not exposed to users. response_headers was empty and never used by any of these helper functions although certain callbacks in the generated code require the placeholder